rust/hyperlane-core/src/traits/mod.rs

```diff
@@ -30,7 +30,7 @@ mod validator_announce;
 #[derive(Debug, Clone, Copy)]
 pub struct TxOutcome {
     /// The txid
-    pub txid: crate::H256,
+    pub txid: crate::TxId,
```

Comment:
Afaict there are no DB implications to making this change (`record_tx_outcome` only stores a `InterchainGasExpenditure`), just want to double check this is the case